src/shogun/evaluation/SigmoidCalibration.h

```diff
@@ -94,6 +94,46 @@ namespace shogun
 		**/
 		virtual CMulticlassLabels* calibrate_multiclass(CMulticlassLabels* predictions);
 
+	 	/** Set maximum number of iterations
+	 	* @param maxiter maximum number of iterations
+	 	*/
+	 	virtual void set_maxiter(index_t maxiter);
```

Comment:
why are these all virtual? you will have a calibration method that is based on Sigmoid?

Reply:
Probably not. Indeed there is no need to make them virtual.